### DISCUSSION TITLE
Having a little issue with settings guide.

### DISCUSSION THREAD

**Discussion by AussiePaul60** [Ghostkeeper/SettingsGuide#107] [General]
I'm new to 3D printing and so I'm finding the Cura settings guide addon very helpful.
 But its not reacting to a right click but it pops up automatically whenever my cursor is over a setting for a few seconds.
 I had installed it via the marketplace. (2.90) 
 I removed that and instead installed the 2.91 version from here but it still does the same thing.
 Windows 10 
Cura 5.1.0 .
The plug in is great.
 I just wanted to pass on that there is a problem, perhaps there is a simple soloution that I have not been able to find?

**Comment by Harry-3D77** [comment]
I have the same problem as AussiePaul60. The auto popup is quite irritating and "in one's face" all the time. I have to uninstall, restart every time I want to work on my start and end g-codes. Before I can change or type something, the popup blocks everything! This is a very helpful addon but definitely needs some update to solve this problem.
Not sure how this messages can get to the developer?

**Comment by 5axes** [comment]
You don't need to uninstall the plugin. Just uncheck in the preference of the plugin the Tooltips option.   
![image](https://user-images.githubusercontent.com/11015345/194523988-f8ef16f5-994c-4653-946d-11ff29c3c8a1.png)

**Reply by AussiePaul60** [reply]
I like the plugin and I don't uninstall  it.
My reason for posting my message was in the hope that the dev or anyone that knows how to configure it might see it and let me know how to turn something off to stop that behaviour.
I have tried to find that settings page that Saxon posted but I can't see how to bring that up.
 I does sound like its worth a try though.

**Comment by Harry-3D77** [comment]
![Cura-setting](https://user-images.githubusercontent.com/112470979/194621453-1c041662-b62c-4734-bc33-616b17f2478f.jpg)

**Comment by Harry-3D77** [comment]
After also searching all over the place I discovered that I had to enable the addon in market place first before I could find the menu option again. 
I still say the UI needs some attention.
